TextFlowExt incompatible with JavaFX 25

The methods defined here: https://github.com/FXMisc/RichTextFX/blob/9f772d9caa7735aaeda74155491b951146d8e33b/richtextfx/src/main/java/org/fxmisc/richtext/TextFlowExt.java#L71

Now override final methods that exist in the base JFX `TextFlow` class: https://github.com/openjdk/jfx/commit/04c5e40cc116cb42150572959b53d1e465700e0e

--------

The exception:
```
Caused by: java.lang.IncompatibleClassChangeError: class org.fxmisc.richtext.TextFlowExt overrides final method javafx.scene.text.TextFlow.getUnderlineShape(II)[Ljavafx/scene/shape/PathElement;
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
        ...
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:490)
	at org.fxmisc.richtext.ParagraphBox.<init>(ParagraphBox.java:98)
        ...
	at org.fxmisc.richtext.GenericStyledArea.<init>(GenericStyledArea.java:811)
	at org.fxmisc.richtext.StyledTextArea.<init>(StyledTextArea.java:33)
	at org.fxmisc.richtext.StyledTextArea.<init>(StyledTextArea.java:45)
	at org.fxmisc.richtext.StyleClassedTextArea.<init>(StyleClassedTextArea.java:21)
	at org.fxmisc.richtext.StyleClassedTextArea.<init>(StyleClassedTextArea.java:34)
	at org.fxmisc.richtext.CodeArea.<init>(CodeArea.java:37)
```

--------

You can observe the failure by using a `CodeArea` with JavaFX 26-ea 

Possible fixes:

- Rename the methods in `TextFlowEx` to maintain compatibility with older JavaFX releases and newer ones
    - The methods added by RichTextFX are package private anyways, so library consumers will see this as an invisible change that fixes JavaFX 25+ incompatibility
- Bump the minimum target to JavaFX 25 and drop support for older releases
    - Library consumers will see this as a patch that requires bumping JavaFX in their own projects, and in turn possibly bumping up to Java 23+ _(As JavaFX has a N-2 version target policy now)_

--- a/richtextfx/src/main/java/org/fxmisc/richtext/TextFlowExt.java
+++ b/richtextfx/src/main/java/org/fxmisc/richtext/TextFlowExt.java
@@ -64,15 +64,16 @@ class TextFlowExt extends TextFlow {
         return rangeShape(from, to);
     }
 
+    /**
+     * Note that this doesn't call super.getUnderlineShape in JavaFX 25+ 
+     */
     PathElement[] getUnderlineShape(IndexRange range) {
-        return getUnderlineShape(range.getStart(), range.getEnd());
-    }
-
-    PathElement[] getUnderlineShape(int from, int to) {
-        return getUnderlineShape(from, to, 0, 0, 0);
+        return getUnderlineShape(range.getStart(), range.getEnd(), 0, 0, 0);
     }
 
     /**
+     * Note that this doesn't call super.getUnderlineShape in JavaFX 25+
+     * 
      * @param from The index of the first character.
      * @param to The index of the last character.
      * @param offset The distance below the baseline to draw the underline.

